.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: c
+before_install: if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew upgrade && brew install gmp; fi
+install: wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-opam.sh
 script: bash -ex .travis-opam.sh
-sudo: false
+sudo: required
```

Comment:
We specifically are using the container infrastructure. There are several reasons for this:
1. Greater amount of RAM (that enables faster builds)
2. Better availability of the workers (builds start much faster)
3. Caching (previously caching was enabled only for the container infrastructure).

I'm not sure, though, whether these benefits are still actual.